core/src/main/scala/flatgraph/storage/Deserialization.scala

```diff
@@ -11,15 +10,30 @@ import java.nio.file.Path
 import java.nio.{ByteBuffer, ByteOrder}
 import java.util.Arrays
 import scala.collection.mutable
+import java.util.concurrent
 
 object Deserialization {
 
-  def readGraph(storagePath: Path, schemaMaybe: Option[Schema], persistOnClose: Boolean = true): Graph = {
+  def readGraph(
+    storagePath: Path,
+    schemaMaybe: Option[Schema],
+    persistOnClose: Boolean = true,
+    executor0: concurrent.ExecutorService = null
```

Comment:
```suggestion
    executor0: Option[concurrent.ExecutorService] = None
```

this doesn't look like it's on the hot path, is it?

Reply:
I'll do that change everywhere, good catch!